modules/islandora_fits/README.md

```diff
@@ -0,0 +1,60 @@
+# Islandora FITS
+Config module to make Islandora aware of FITS microservice
+
+## Installation
+#### Install this module
+Install and enable this module in the usual way.  On installation the module will
+add a context causing the creation of A FITS media when an Original File media is ingested; however, this process is
+predicated on the existence of an `islandora_media_use` term with an external URI of `https://projects.iq.harvard.
+edu/fits`--the `islandora_fits_tags` migration might be executed to create such a term.
+
+#### Install FITS Webservice
+FITS XMLs are generated from an easily installed web service.
+Get the latest fits.zip and fits.war from https://projects.iq.harvard.edu/fits/downloads
+(on my box I had to install a missing zip library with
+‘sudo apt-get install php7.1-zip’)
+
+Install following their instructions.
+Copy the `.war` file to your webapps directory and test.
+Edit the `catalina.properties` file on the Drupal server by adding the
+following two lines to the bottom of the file:
+
+```properties
+fits.home=/\<path-to-fits>/fits
+shared.loader=/\<path-to-fits>/fits/lib/*.jar
+```
+
+Restart Tomcat and test with:
+```bash
+curl -k -F datafile="@/path/to/myfile.jpg" http://example.com:8080/fits/examine
+```
+(note: the ‘@’ is required.)
+
+#### Installing Microservice
+Get code from https://github.com/roblib/CrayFits and install.  This code can live anywhere, including an external server,
```

Comment:
Making reference to the roblib org specifically; should be updated to point at however it gets moved over to the IF, whenever it happens (not sure if there's an issue/PR for it, yet?).

Reply:
there's an [issue](https://github.com/Islandora/Crayfish/issues/173) but no PR. I saw there's currently[ work being done for PHP 8.1](https://github.com/roblib/CrayFits/pull/10) and figured it'd be good to wait till that's in.